image/codecs/indeo3.cpp

```diff
@@ -468,18 +491,19 @@ void Indeo3Decoder::decodeChunk(byte *cur, byte *ref, int width, int height,
 			}
 		}
 
-		cur_frm_pos = cur + width * strip->ypos + strip->xpos;
+		cur_frm_pos = (uint32 *)(cur + width * strip->ypos + strip->xpos);
```

Comment:
Will that work correctly on BE systems?

Reply:
It should be OK. In the existing code, the same cast would be made later on, and the `FROM_LE_32` is used if the values in the array are modified.